src/Nethermind/Nethermind.Consensus/Processing/OneTimeProcessor.cs

```diff
@@ -39,17 +39,7 @@ public Task StopAsync(bool processRemainingBlocks = false)
         {
             lock (_lock)
             {
-                Block result;
-                try
-                {
-                    result = _processor.Process(block, options, tracer);
-                }
-                finally
-                {
-                    _worldState.Reset();
```

Comment:
Previously this was a `ReadOnlyDbProvider.ClearTempChanges()`. It does not actually do anything as the `AsReadOnly` have false as its parameter except for code db. Meaning only code db have buffer.

Reply:
Can it break something in RPC? like eth_call or tracing?

Reply:
Not sure, so I'm gonna be conservative and revert back to what it was.